Tasks/AzureCloudPowerShellDeployment/Utility.ps1

```diff
@@ -0,0 +1,136 @@
+﻿function Get-SingleFile($files, $pattern)
```

Comment:
I thought about this a bit more. I'm somewhat on the fence whether this is a fragile assumption or not. I think I'm leaning toward not now. Ultimately the Find-VstsFiles function should always return objects by writing them into the output pipeline. When the output is captured and more than one object is written, the `-is [system.array]` test appears to hold true (tested on powershell 5 only).

The thing that makes me worry about this assumption is that the books are always intentionally vague as to when powershell uses arrays versus arraylist, etc.

Ultimately I think the approach taken here is OK probably. It makes me a little bit nervous, but it's probably fine.